test/OptionsTest.cs

```diff
@@ -85,15 +85,17 @@ public void GetKeyHandlers()
                     Assert.Equal("Home", handler.Key);
                 }
             }
+        }
+
+        [SkippableFact]
+        public void GetKeyHandlers_Emacs()
+        {
+            System.Collections.Generic.IEnumerable<Microsoft.PowerShell.KeyHandler> handlers;
 
             TestSetup(KeyMode.Emacs);
             
             handlers = PSConsoleReadLine.GetKeyHandlers(Chord: new string[] { "ctrl+x" });
-            Assert.NotEmpty(handlers);
-            foreach (var handler in handlers)
-            {
-                Assert.Equal("Ctrl+x", handler.Key);
-            }
+            Assert.Empty(handlers);
```

Comment:
As far as I can see, this is the only behaviour change. Previously to this PR, a partial chord would return `ChordFirstKey` as its handler.

This PR eliminates this indirection and does not return any handler for partial chords. Only handlers at the leaf are returned.